samples/Interop45/Compat/CompatRole.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation, Inc. All rights reserved.
+// Licensed under the MIT License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.Identity.EntityFramework;
+
+namespace Microsoft.AspNet.Identity.Compat
+{
+    public class IdentityRole : IdentityRole<string, IdentityUserRole>
+    {
+        /// <summary>
+        ///     Constructor
+        /// </summary>
+        public IdentityRole()
+        {
+            Id = Guid.NewGuid().ToString();
+        }
+
+        /// <summary>
+        ///     Constructor
+        /// </summary>
+        /// <param name="roleName"></param>
+        public IdentityRole(string roleName)
+            : this()
+        {
+            Name = roleName;
+        }
+    }
+
+    public class IdentityRole<TKey, TUserRole> : EntityFramework.IdentityRole<TKey, TUserRole>
+        where TUserRole : IdentityUserRole<TKey>
+    {
+        /// <summary>
+        ///     Normalized role name
+        /// </summary>
+        public virtual string NormalizedName { get; set; }
+
+        /// <summary>
+        ///     Concurrency stamp 
+        /// </summary>
+        public virtual string ConcurrencyStamp { get; set; } = Guid.NewGuid().ToString();
```

Comment:
Probably not. I copied this from codeplex as is. I'm guessing it's broken in the same way that normalized user name is.